WcaOnRails/app/models/registration_payment.rb

```diff
@@ -6,4 +6,8 @@ class RegistrationPayment < ActiveRecord::Base
            as: "amount",
            allow_nil: true,
            with_model_currency: :currency_code
+
+  def amount_available_for_refund
+    amount_lowest_denomination + RegistrationPayment.where(registration_payment_id: id).sum("amount_lowest_denomination")
```

Comment:
Yeah when I updated the column name it crossed my mind it wasn't a great name. I should have given it a bit more thought.